src/components/list/listcontainer.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class='uk-panel uk-width-1-1'>
+    <ul class='uk-list uk-list-divider'>
+      <list-item
+        v-for='item in items'
+        :key='item.uid'
+        :uid='item.uid'
+        :editor='EditorClass'
+        @delete-item='onDeleteItem'
+        @change-item='onChangeItem'
+      ></list-item>
+    </ul>
+    <div class='uk-panel uk-margin'>
+      <button class="uk-button uk-button-primary uk-float-right"
+      @click="createItem">Add Item</button>
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import ListItem from './listitem.vue'
+import ModalDialog from '../modaldialog.vue'
+
+export default {
+  name: 'list-container',
+  components: {
+    ListItem,
+  },
+  props: {
+    editor: {
+      type: Object,
+      required: true
+    },
+  },
+  data () {
+    return {
+      currentItem: {},
+      ModalDialogClass: Vue.extend(ModalDialog),
+      EditorClass: null,
+      dialogInstance: null,
+      items: {},
+    }
+  },
+  methods: {
+    createItem () {
+      this.currentItem = {}
```

Comment:
Is it worth having a method on the store that gives you an empty data source object that you can then pass in to the editor?

Reply:
I don't think so, at least not yet. There aren't good defaults.